client/src/pages/manager/login/Login.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState, useRef } from "react"
+import LoginService from "../../../services/auth/index"
+import './login.css'
+
+// TODO: implement magic login logic 
+function Login() {
+
+  const [email, setEmail] = useState("")
```

Comment:
Run prettier. Seems like a lot of your indents are formatted as spaces instead of true indents, and there's an extra space on the bottom.